src/chat.py

```diff
@@ -1,15 +1,18 @@
 import os
 from llama_cpp import Llama
+from langchain.embeddings.sentence_transformer import SentenceTransformerEmbeddings
+from langchain.vectorstores import Chroma
 
 class Chat:
 
-    n_ctx = 2048
+    n_ctx = 4096
+    
 
     def __init__(self) -> None:
         self.chat_history = [
                 {"role": "system", "content": """You are a helpful assistant that is comfortable speaking
-                with C level executives in a professional setting."""},
-                ]
+                 with C level executives in a professional setting."""},]
+        self.embeddings = SentenceTransformerEmbeddings(model_name="BAAI/bge-base-en-v1.5")        
```

Comment:
Is this going to download a model for embedding unless it already exists in the the cache?

Reply:
Can we have our embedding model part of the image in such a way that its not coming from the internet every time we instantiate a Chat object? Like read it from a local file instead that we downloaded a head of time?

Reply:
Yes certainly, It's gonna add ~500Mib to the image